literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml

```diff
@@ -0,0 +1,102 @@
+# This file describes a figure from a publication containing electrochemical data.
+# For the `echemdb/website` project, the source, curator and electrochemical system is mandatory.
+# Figure description is optional. It is created automatically by the cv module of svgdigitizer
+
+source:
+    version: 1
+    doi: 10.1021/jp9600064 # only the number without preceeding doix or https
+    bib: wandlowski_1996_structural_10277 # AuthorName_YYYY_FirstWordTitle_pageNr. No prepositions. Fingers crossed that there is no duplicate.
+    figure: 1a # depending on the subfigure labels use 1, 1b, or 1_3
+    curve: dashed # use a unique short description to identify the curve in the figure, i.e.. label, color, etc
+    
+curator:
+    version: 1
+    name: Markus Schilling
+    orcid: https://orcid.org/0000-0003-2684-3711
+    digitized: 2021-11-04 # YYYY-MM-DD
+    
+figure description: #This section will be updated by module `CV`
+    version: 1
+    linked measurements: in-situ surface X-ray # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data

```

Comment:
"[in situ](https://en.wikipedia.org/wiki/In_situ)" is usually written without a dash. (No dash in this latin wording)

Reply:
Is surface X-Ray abbreviated as SXRD in the manuscript?

Reply:
The description was taken from the first sentence of the abstract of that paper (with dash). I am fine with both.
SXRD is single crystal XRD for me, which is different in that case. SXRD is not used in the paper.

How should I change it?

Reply:
So it seems that the authors write "in situ surface x-ray diffraction" in the introduction and "surface x-ray scattering - SXS" in the manuscript. So probably you should simply change it to SXRD, which is commonly used nowadays.

Reply:
changed to SXRD